### PR TITLE
Removes Unnecessary Tools

### PR DESCRIPTION
**Pull request by redreceipt**
**IMPORTANT: If this is for testing code in our npm dependencies, you should be pointing your PR at the `canary` or `next` branch. `master` should never rely on unreleased code.**

### CODE REVIEW

File: apolloschurchapp/app.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (vinnyjth) [inline apolloschurchapp/app.json:1]: I think we still need this file.

Comment (vinnyjth) [reply apolloschurchapp/app.json:1]: https://github.com/react-native-community/cli/issues/1113

File: apolloschurchapp/appcenter-post-clone.sh [hunk 2]
```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
```
Comment (vinnyjth) [inline apolloschurchapp/appcenter-post-clone.sh:1]: We should probably leave these in for now. I don't want them to be accidentally deleted from our client repos when they update and cause confusion and force a premature migration to Github Actions.

File: apolloschurchapp/package.json [hunk 3]
```diff
@@ -12,16 +12,11 @@
     "setup": "./scripts/init.sh",
     "android": "SIM=\"$($HOME/Library/Android/sdk/emulator/emulator -list-avds | sed -n 1p)\" && $HOME/Library/Android/sdk/emulator/emulator -avd \"${SIM}\" & react-native run-android --appId com.differential.apollos",
     "codecov": "cat ./coverage/lcov.info | codecov",
-    "fixlint": "eslint ./src --fix",
```
Comment (vinnyjth) [inline apolloschurchapp/package.json:15]: nice ;)

File: apolloschurchapp/app.json (hunk 1 above)
Comment (redreceipt) [reply apolloschurchapp/app.json:1]: don't think we do. it came from expo, was just to pull the name from inside the index.js file (which ours doesn't do)

File: apolloschurchapp/appcenter-post-clone.sh (hunk 2 above)
Comment (redreceipt) [reply apolloschurchapp/appcenter-post-clone.sh:1]: fair. I could a separate PR to remove these pointed to next

File: apolloschurchapp/app.json (hunk 1 above)
Comment (vinnyjth) [reply apolloschurchapp/app.json:1]: I ran rn init recently, let me dig up what it included....

Comment (vinnyjth) [reply apolloschurchapp/app.json:1]: okay - so the default react-native app does create that app.json.

Comment (redreceipt) [reply apolloschurchapp/app.json:1]: ah. so according to that issue, it's created on init but not used anymore. I can put it back, it's just another file churches *should* rename even though it seems unnecessary

Comment (redreceipt) [reply apolloschurchapp/app.json:1]: was trying to avoid as much renaming as possible

Comment (vinnyjth) [reply apolloschurchapp/app.json:1]: that's fair - none of our churches have ever renamed it to my knowledge so I'm fine with deleting.

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**